contracts/TokenVault.sol

```diff
@@ -178,6 +189,36 @@ contract TokenVault is Ownable, Recoverable {
     return token.balanceOf(address(this));
   }
 
+  /// @dev Check how many tokens "investor" can claim
+  /// @param investor Address of the investor
+  /// @return uint How many tokens the investor can claim now
+  function getCurrentlyClaimableAmount(address investor) public constant returns (uint claimableAmount) {
+    uint maxTokensLeft = balances[investor] - claimed[investor];
+
+    if (now < freezeEndsAt) {
+      return 0;
+    }
+
+    if (tokensPerSecond > 0) {
+      uint previousClaimAt = lastClaimedAt[investor];
+      uint maxClaim;
+
+      if (previousClaimAt == 0) {
+        previousClaimAt = freezeEndsAt;
+      }
+
+      maxClaim = (now - previousClaimAt) * tokensPerSecond;
```

Comment:
Similarly, here I would rename `maxClaim` to something like `maxCurrentClaimableAmount`. This could make the following `if` easy to understand with: `if (maxCurrentClaimableAmount > totalClaimableAmount) {`